pkg/provisioner/crdprovisioner.go

```diff
@@ -1200,6 +1250,17 @@ func waitForAttachVolumeFunc(obj interface{}, objectDeleted bool) (bool, error)
 	return false, nil
 }
 
+func waitForAttachVolumeCompleteFunc(obj interface{}, objectDeleted bool) (bool, error) {
```

Comment:
np: since this method is specific to Primary attachments, only returns true for success and doesn't do the waiting maybe the name could be something like `isPrimaryAttachmentSuccessful` or `isPrimaryAttachSuccessful`.